### ISSUE TITLE
Magic frozen string comment breaks component

### ISSUE THREAD

**Issue by kucho** [joeldrapper/phlex#60]
Hello, 

I started migrating some ViewComponent files to Phlex but I am getting "can't modify frozen String" whenever I try to include a class attribute to an anchor tag. 

```ruby
# app/frontend/navbar/component.rb

# frozen_string_literal: true

module Navbar
  class Component < Phlex::Component
    def template
      nav.navbar.bg_base_100 do
        a("Get started", href: Current.rodauth.login_path, class: "btn")
      end
    end
  end
end
```

After checking out this [file](https://github.com/joeldrapper/phlex/blob/main/lib/phlex/tag.rb#L23), I realized that I needed to remove the magic comment. I think it should be added in the README or are they any plans to support it?

Also, I think it would be nice to have a section to show how to embed a Phlex component in an erb template for starters. It's getting very late over here and  It took me more than I am willing to admit to come up with: `<%= render html: Navbar::Component.new.call.html_safe %>` :sleepy: :joy:

**Comment by joeldrapper** [comment]
Wow, thanks for helping us find this bug. Looks like we need to dup the classes String before modifying it.

**Comment by joeldrapper** [comment]
Hey @kucho, I pushed a fix in #61. Should be merged to main soon, but you can always point your Gemfile to that branch for now. Do check out the `#component` method in #36, which you should be able to copy to your `ApplicationHelper` in Rails. We're aiming to completely replace ActionView, but of course there will need to be a transition period where you’ll want to render Phlex components in ActionView templates and maybe ActionView partials in Phlex components.
